src/internal/cache/cache.go

```diff
@@ -45,7 +45,7 @@ type Cache interface {
 	CableCurrent() *int64
 	// CableAlwaysLocked returns state of cable always locked parameter.
 	CableAlwaysLocked() bool
-	IsSessionFinished() bool
+	HasOngoingChargingSession() bool
```

Comment:
We don't need a separate field for that, as we can probably deduce that value based on data we already have in the cache.